e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1

```diff
@@ -283,11 +287,16 @@ $iotHubX509SelfSignedDeviceCert = New-SelfSignedCertificate `
     -TextExtension @("2.5.29.37={text}1.3.6.1.5.5.7.3.2") `
     -HashAlgorithm "$certificateHashAlgorithm" `
     -CertStoreLocation "Cert:\LocalMachine\My" `
-    -NotAfter (Get-Date).AddYears(2)
+    -NotAfter (Get-Date).AddYears(2) `
+    -Signer $intermediateCert2
+
+Export-PFXCertificate -cert $iotHubX509ChainDeviceCert -filePath $iotHubX509ChainDevicPfxPath -password $iotHubCredentials.Password | Out-Null
+$iothubX509ChainDevicePfxBase64 = [Convert]::ToBase64String((Get-Content $iotHubX509ChainDevicPfxPath -AsByteStream));
```

Comment:
It looks like this section was misplaced during the merge. The `$iotHubX509SelfSignedDeviceCert` is not expected to have a `-Signer`, and `$iotHubX509ChainDeviceCert` is initialized below in L303.

https://github.com/Azure/azure-iot-sdk-csharp/blob/f16f4182231aa093274c33728c53ff72dbb76d6a/e2e/test/prerequisites/E2ETestsSetup/e2eTestsSetup.ps1#L278-L305